api/applications_api/serializers.py

```diff
@@ -667,6 +665,10 @@ class Meta:
         read_only_fields = ["id", "timestamp"]
 
     def create(self, validated_data):
+        request = self.context.get("request")
         instance = ApplicationEventStatus(**validated_data)
+        if request:
+            instance.user = request.user
+        instance.user = self.request.user
```

Comment:
Isn't this already set in the previous line?

Reply:
good spot :+1:  that's poor cleanup by me.

Reply:
should be fixed now